plumbing/object/change.go

```diff
@@ -61,6 +70,12 @@ func (c *Change) Files() (from, to *File, err error) {
 	return
 }
 
+func isInvalidFileMode(t TreeEntry) bool {
```

Comment:
Use `Filemode.IsFile()`, that's what @santi and me agreed when I answered https://github.com/src-d/go-git/issues/317.